common/log/test_logger.go

```diff
@@ -0,0 +1,39 @@
+package log
+
+import (
+	"fmt"
+	"go.temporal.io/server/common/log/tag"
+	"testing"
+)
+
+type testLogger struct {
+	t *testing.T
+}
+
+func (l testLogger) Debug(msg string, tags ...tag.Tag) {
```

Comment:
`(l testLogger)` -> `(l *testLogger)`

Reply:
btw, why not using this?
https://github.com/temporalio/temporal/blob/v1.14.3/common/log/zap_logger.go#L57

Reply:
not related to this pr, but shall we use https://pkg.go.dev/go.uber.org/zap/zaptest#NewLogger for test logger?

Reply:
I was not aware of `zaptest.NewLogger`. That's kind of what I was looking for, but I need a temporal `log.Logger`, not a zap.Logger. It seems like there should be a way to wrap a zap.Logger as a log.Logger, but I'm not seeing it.

Reply:
D'oh. NewZapLogger. I'm good now.